index.html

```diff
@@ -1286,9 +1286,7 @@ <h2>DID Subject</h2>
 However, the fully resolved <a>DID document</a> always contains a valid
 <code><a>id</a></code> property. The value of <code><a>id</a></code> in the
 resolved <a>DID document</a> MUST match the <a>DID</a> that was
-resolved, or be populated with the equivalent canonical <a>DID</a>
-specified by the <a>DID method</a>, which SHOULD be used by the resolving
```

Comment:
The canonical behavior is still valuable.

https://github.com/w3c/did-core/issues/421#issuecomment-705257340

Reply:
I agree a canonical property is a good idea, and would support an additional separate PR that defines one.

Reply:
@csuwildcat cool - I think the change to canonical language belongs in a different PR rather than this one.

Reply:
The other option here is to just add a separate `canonicalId` property and not this currently proposed one, but the language would need to mirror this property's language, not force a switch between them.

Reply:
@csuwildcat We want the ability to signal both the equivalence assertion and the preferred DID (and when combined is the canonical DID).

Yes - our scenario can be handled via a `canonicalId` property.

Reply:
I think the combination of `deterministicSameAs` and a preferred ID semantics is the replacement to the canonical DID capability. As such, it would be sensible to have this PR define both and replace this text.

Reply:
Thanks for the update - my comment is resolved.